estela-web/src/components/ProjectListPage/index.tsx

```diff
@@ -60,8 +67,15 @@ export class ProjectListPage extends Component<unknown, ProjectsPageState> {
             authNotification();
         } else {
             const data = await this.getProjects(1);
-            const projects: Project[] = data.data;
-            this.setState({ projects: [...projects], count: data.count, current: data.current, loaded: true });
+            const projectData: ProjectList[] = data.data.map((project: Project, id: number) => {
+                return {
+                    name: project.name,
+                    pid: project.pid,
+                    role: "Admin",
+                    key: id,
+                };
+            });
+            this.setState({ projects: [...projectData], count: data.count, current: data.current, loaded: true });
```

Comment:
Probably you have conflict with this PR https://github.com/bitmakerla/estela/pull/101 if you merge both.